docs/ Screening/continuous-monitoring.md

```diff
@@ -0,0 +1,27 @@
+# Continuous Monitoring
+
+## Continuous Criminal Monitoring
+Turn’s Continuous Criminal Monitoring service will help our customers to keep a close eye on any criminal activity with active employees.
+
+Upon enrollment, Continuous Criminal Monitoring will deliver fresh background check results including the alerted county due to arrest events that occur with monitored individuals.
+
+1.  Monitor all arresting events at State/County Booking Facilities.
+2.  Run a background check at the county criminal level on the alerted individuals, and
+3.  Deliver background check results and adjudication determinations (if applicable) based on those results.
+
+This feature allows users to request the following actions via API:
+
+\t\- Enroll: Add to the Continuous Criminal Monitoring service.
```

Comment:
I don't think these are rendering correctly:
<img width="1067" alt="image" src="https://github.com/turnhq/stoplight/assets/1648963/26966334-2015-4b49-9b76-cddada3c538e">

https://apidoc.turn.ai/docs/turn-api/branches/SCREEN-3692-api-documentation-for-cm-enrollment-requests-recheck-responses/845f669c459e2-continuous-monitoring